### ISSUE TITLE
Missing right pane for issue options (labels, milestones, pipeline..) when opening new issues.

### ISSUE THREAD

**Issue by apirri** [ZenHubIO/support#808]
I had recently been added as a member to an organization but when attempting to open a new issue I am not able to add labels, milestones, pipeline, epic... because the right options pane is not available. I am having this issue in both Chrome and Firefox. The odd thing is that I do have the right pane available in other organizations I am owner or member of.

![image](https://cloud.githubusercontent.com/assets/20192900/16427839/e47a630c-3d34-11e6-85f0-1a66fb72a679.png)

**Comment by aupright** [comment]
Hey @apirri - looks like this could be a permissions issue in GitHub. Can you confirm that you have write access for the repo that you're trying to view the right-hand panel within?

**Comment by apirri** [comment]
Hey Aaron,

        Thanks for the quick response! This appears to be what the issue 
is, they had the default member access as 'read'. When I was made to owner 
this issue went away but when new people were added they had the same 
issue. I updated it to write and am waiting to confirm it resolves the 
issue.

Thanks,
Tony Pirri
Advisory Project Manager
Performance Engineering, Cloud Infrastructure Services
Email: apirri@us.ibm.com
